CMakeLists.txt

```diff
@@ -4,8 +4,278 @@ project(libtorrent)
 set (SOVERSION "10")
 set (VERSION "1.2.0")
 
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
I think `target_compile_features(torrent-rasterbar PUBLIC cxx_std_11)` is the current preferred way of specifying that C++11 is required.

Reply:
that's nice, didn't know about it, and it propagates the requirement